_quarto.yml

```
@@ -0,0 +1,38 @@
+project:
+  type: website
+
+website:
+  title: "NFIDD"
+  site-url: https://nfidd.github.io/nfidd/
+  description: "Nowcasting and forecasting infectious disease dynamics"
+  image: images/logo.png
+  favicon: images/favicon-32x32.png
+  open-graph: true
+  comments:
+    giscus:
+      repo: nfidd/nfidd
+      category: 'Website discussions'
+      mapping: title
+      reactions-enabled: true
```

later we deactivate them everywhere? Perhaps leave discussions to the github site (open for discussion)

```suggestion
  comments: false
```

---

We don't have them turned off everywhere we just have them turned off on not sessions. This is nice as the comments are posts on the github discussion. That being said I see why we might not want comments so it's definitely a judgement call.